src/mmrelay/k8s_utils.py

```
@@ -0,0 +1,534 @@
+
+    # Check if file exists
+    if not os.path.isfile(configmap_path):
+        logger.error(f"ConfigMap file not found: {configmap_path}")
+        return False
+
+    # Load ConfigMap YAML
+    try:
+        with open(configmap_path, "r", encoding="utf-8") as f:
+            configmap_content = f.read()
+    except (OSError, UnicodeDecodeError) as e:
+        logger.error(f"Error reading ConfigMap file: {e}")
```

Ruff (0.14.13)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)